packages/webdriver-utils/package.json

```diff
@@ -0,0 +1,28 @@
+{
```

Comment:
ref - [sdk-utils/package.json](https://github.com/percy/cli/blob/8de18f03aa611d539d45b40ba3af5accf237a78b/packages/sdk-utils/package.json)

Reply:
Is there something specific to look for over here ?

Reply:
why are we reducing test coverage to 75%? lets have it 100% like other packages

Reply:
```json
  "publishConfig": {
    "access": "public"
  },
```

export both commonjs and import format

Eg:
```
 "exports": {
    ".": {
      "node": "./dist/index.js",
      "default": "./dist/bundle.js"
    },
    "./test/helpers": {
      "node": "./test/helpers.js",
      "default": "./test/client.js"
    }
  }
```

verify ^ by `requiring` and `importing` the package

Reply:
We won't be exporting to user, so currently sticking with esmodules.